app/src/test/java/uk/ac/ebi/atlas/experimentpage/tabs/ExperimentPageContentServiceIT.java

```diff
@@ -66,7 +66,6 @@ void populateDatabaseTables() {
         var populator = new ResourceDatabasePopulator();
         populator.addScripts(
                 new ClassPathResource("fixtures/experiment-fixture.sql"),
-                new ClassPathResource("fixtures/scxa_tsne-fixture-alt.sql"),
```

Comment:
haha..you have removed `scxa_tsne` fixture references in this PR. I gave a comment in the previous [PR](https://github.com/ebi-gene-expression-group/atlas-web-core/pull/64) while revewing.